src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java

```diff
@@ -79,7 +79,7 @@ public String getDescription() {
         }
 
         public String getIconFilePathPattern() {
-            return "plugin/workflow-images/:size/pipelinemultibranchproject.png";
+            return "plugin/workflow-multibranch/:size/pipelinemultibranchproject.png";
```

Comment:
In the original PR was fine https://github.com/jenkinsci/pipeline-plugin/pull/376/files#diff-fc4cf5ce663b563407ec5283d4866c34R82

I found the problem. I had to do a replace of `multibranch` to adapt the patch file :sob: